ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/AttributeAnalyzer/AttributeAnalyzerVisitor/JavaAttributeVisitor.java

```diff
@@ -0,0 +1,40 @@
+package edu.usb.argos.ASTProcessor.AttributeAnalyzer.AttributeAnalyzerVisitor;
```

Comment:
Please change your folder structure and try to follow the structure according to the `feature/cs-021-7/method-parser-visitor` branch, move your classes according to that structure.